frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -65,7 +65,7 @@ class TripsContainer extends React.Component {
    *
    * @override
    */
-  async componentDidMount() {
+  async componentDidUpdate() {
```

Comment:
This looks like a change from a different PR?

Reply:
It is not. This was needed in order to update the TripsContainer whenever the trip view (change tabs) is updated. This also caused #134 but that is fixed in the following PR.